src/wavesurfer.js

```diff
@@ -289,6 +289,17 @@ var WaveSurfer = {
         return this.isMuted;
     },
 
+    /**
+     * Get the list of current set filters as an array.
+     *
+     * Filters must be set with setFilters method first
+     */
+    getFilters: function() {
+        if (typeof this.backend.filters !== 'undefined') {
```

Comment:
It will return `undefined` in your current code as well. There's no `else`-clause, so the function returns nothing, i.e. `undefined`.